### DISCUSSION TITLE
Code - Settings Sync doesn’t work

### DISCUSSION THREAD

**Discussion by sonnypgs** [blinksh/blink#1505] [Support] [answered]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

Blink v15.0.8.542
iPadOS 16.0 (20A5312j)

### Describe the bug

Settings Sync is not working within the Code application. Clicking on `Turn on Settings Sync` works - it’s opening the select dialog. After clicking on `Sign in & Turn on` nothing happens. 

I tried the same scenario on vscode.dev, where the sync works without problems.

![IMG_0227](https://user-images.githubusercontent.com/1431300/178800581-6abbce9a-2c11-4c45-b030-8d9eedd08a0a.png)

**Comment by carloscabanero** [comment]
This is controlled through GitHub syncing. I have my settings synced this way, so “It should just work (TM)”. I know VSCode was acting funny the last days, are you still not able to sync?

**Reply by sonnypgs** [reply]
Thanks for your reply @carloscabanero :) Just tried it again, sadly still the same. I suppose it has something to do with the iPadOS16 beta - will try again with iPadOS15 and report back.

**Comment by wmorgue** [comment]
Same for me:

```text
[2022-08-26 09:50:12.311] [settingssync] [info] Did reset the local sync state.
[2022-08-26 09:53:28.382] [settingssync] [info] Request failed https://vscode-sync.trafficmanager.net/v1/resource/extensions/latest
[2022-08-26 09:53:28.433] [settingssync] [error] h@https://main.vscode-cdn.net/stable/e4503b30fc78200f846c62cf8091b76ff5547662/out/vs/workbench/workbench.web.main.js:799:3522435
l@https://main.vscode-cdn.net/stable/e4503b30fc78200f846c62cf8091b76ff5547662/out/vs/workbench/workbench.web.main.js:799:3522684
@https://main.vscode-cdn.net/stable/e4503b30fc78200f846c62cf8091b76ff5547662/out/vs/workbench/workbench.web.main.js:806:4024439
asyncFunctionResume@[native code]
@[native code]
promiseReactionJobWithoutPromise@[native code]
promiseReactionJob@[native code]
[2022-08-26 09:53:28.436] [settingssync] [info] Settings: Stopped synchronizing settings.
[2022-08-26 09:53:28.438] [settingssync] [info] Keybindings: Stopped synchronizing keybindings.
[2022-08-26 09:53:28.439] [settingssync] [info] Snippets: Stopped synchronizing snippets.
[2022-08-26 09:53:28.441] [settingssync] [info] Tasks: Stopped synchronizing tasks.
[2022-08-26 09:53:28.442] [settingssync] [info] GlobalState: Stopped synchronizing globalstate.
[2022-08-26 09:53:28.443] [settingssync] [info] Did reset the local sync state.
```

After 15 minutes, I restarted blink, reopened the repository, and tried to sync the settings. Only on the fourth attempt was the synchronization successful.

![Screenshot 2022-08-26 at 10 04 20](https://user-images.githubusercontent.com/24982084/186842903-690cd119-8125-4487-8e8f-81cc800a6913.png)



iPadOS 15 | Blink+ Plan v15.0.9.544 from App Store.

**Reply by carloscabanero** [reply]
We are keeping an eye on this. Not sure the issue is from our side, as actually we are now a full browser and should have no problem reaching out to URLs all around the place. Is the Codespace over regular Safari syncing without issue? Is vscode.dev also syncing with no problem?

**Reply by wmorgue** [reply]
Indeed, it looks like a network error on my side. Codespace and vscode.dev work fine in Safari.

**Comment by rushvora** [comment, marked as answer]
@sonnypgs this might be a silly question.. but are you using a keyboard with a trackpad, and are you using the trackpad cursor to select the `Sign in & Turn on` button? I was doing that, and I had the same issue as you, nothing happened after doing so. But then I randomly decided to touch the button with my finger instead, and it worked.

**Reply by nurumaik** [reply]
No way, this really helped. How did you even figured

**Reply by Edditoria** [reply]
I had the same problem. Fingering helps. How did even figured +1 LOL
